invenio_saml/invenio_accounts/utils.py

```diff
@@ -107,6 +107,25 @@ class RegistrationForm(_security.confirm_register_form):
     return RegistrationForm(*args, **kwargs)
 
 
+def _get_csrf_disabled_param():
+    """Return the right param to disable CSRF depending on WTF-Form version.
+
+    From Flask-WTF 0.14.0, `csrf_enabled` param has been deprecated in favor of
+    `meta={csrf: True/False}`.
+    """
+    import flask_wtf
+    from pkg_resources import parse_version
+    supports_meta = parse_version(
+        flask_wtf.__version__) >= parse_version("0.14.0")
+    return dict(meta={'csrf': False}) if supports_meta else \
+        dict(csrf_enabled=False)
+
+
+def create_csrf_disabled_registrationform():
```

Comment:
In `invenio-oauthclient` master branch, we made the registration form customizable, in case a user wants to add extra field (e.g. accept terms and conditions).
Did you have issues using those ones?